### PR TITLE
skip crictl install if present, only download one version

### PR DESCRIPTION
**Pull request by alexeldeib**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/cse_install.sh [hunk 1]
```diff
@@ -103,9 +103,8 @@ downloadCrictl() {
 installCrictl() {
     CPU_ARCH=$(getCPUArch)  #amd64 or arm64
     currentVersion=$(crictl --version 2>/dev/null | sed 's/crictl version //g')
-    local CRICTL_VERSION=${KUBERNETES_VERSION%.*}.0
-    if [[ ${currentVersion} =~ ${CRICTL_VERSION} ]]; then
-        echo "version ${currentVersion} of crictl already installed. skipping installCrictl of target version ${CRICTL_VERSION}"
+    if [[ "${currentVersion}" == "" ]]; then
```
Comment (chmill-zz) [inline parts/linux/cloud-init/artifacts/cse_install.sh:106]: This reads like we're not installing if currentVersion is empty. Which doesn't seem like the goal. Do we want != ?

Comment (alexeldeib) [reply parts/linux/cloud-init/artifacts/cse_install.sh]: oops. flipped

File: parts/linux/cloud-init/artifacts/cse_install.sh [hunk 2]
```diff
@@ -103,9 +103,8 @@ downloadCrictl() {
 installCrictl() {
     CPU_ARCH=$(getCPUArch)  #amd64 or arm64
     currentVersion=$(crictl --version 2>/dev/null | sed 's/crictl version //g')
-    local CRICTL_VERSION=${KUBERNETES_VERSION%.*}.0
```
Comment (alexeldeib) [inline parts/linux/cloud-init/artifacts/cse_install.sh:106]: this isn't even getting called now but w.e.

File: vhdbuilder/packer/install-dependencies.sh [hunk 3]
```diff
@@ -172,6 +172,7 @@ if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
       CRICTL_VERSIONS=""
       if [[ ${CRICTL_VERSIONS_STR} != null ]]; then
         CRICTL_VERSIONS=$(echo "${CRICTL_VERSIONS_STR}" | jq -r ".[]")
+        CRICTL_VERSIONS=$(echo -e "$CRICTL_VERSIONS" | tail -n 2 | head -n 1 | tr -d ' ')
```
Comment (alexeldeib) [inline vhdbuilder/packer/install-dependencies.sh:175]: TODO: more cleanup

**Review by chmill-zz — APPROVED** [review]

**Pull request merged**